tools/swift-api-digester/swift-api-digester.cpp

```diff
@@ -674,6 +674,23 @@ static bool isOwnershipEquivalent(ReferenceOwnership Left,
   return false;
 }
 
+static void detectReordering(SDKNodeDecl *L, SDKNodeDecl *R, SDKContext &Ctx) {
+  if (L->isImplicit())
```

Comment:
It's not correct to ignore implicit declarations. For example, I think the underlying storage of a lazy property is implicit. Please test re-ordering lazy properties.